ci-operator/step-registry/ipi/deprovision/deprovision/powervc/deprovision/ipi-deprovision-deprovision-powervc-deprovision-commands.sh

```
@@ -1,153 +1,432 @@
+	# Kill any child processes
+	local children
+	children=$(jobs -p 2>/dev/null || true)
+	if [[ -n "${children}" ]]; then
+		log_info "Killing child processes: ${children}"
+		# shellcheck disable=SC2086
+		kill ${children} 2>/dev/null || true
+		wait 2>/dev/null || true
+	fi
+	
+	return "${exit_code}"
+}
```

Shellcheck (0.11.0)

[info] This function is never invoked. Check usage (or ignored if invoked indirectly).

(SC2329)